app/src/main/java/com/example/karam/LaborerStatusPage.java

```diff
@@ -0,0 +1,74 @@
+package com.example.karam;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ListView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class LaborerStatusPage extends AppCompatActivity {
+    private static String[] languages = {"English",
```

Comment:
make all string literal final

Reply:
Also when this is changed to private static final, update variable name to LANGUAGES . static final variables are UPPER_SNAKE_CASE

Reply:
All the static strings will go to res/values/strings.xml. I will make these changes when I write the code for the business logic because most of the strings that I have currently used are for dummy values.